amp-script-tutorial/README.md

```diff
@@ -0,0 +1,29 @@
+# amp-script-tutorial
```

Comment:
I'm worried about using component naming dashes like this.  Might cause confusion.

Reply:
It is a little confusing, isn't it? But I'm just following the naming convention used in this repo, and in other projects' README's.

Reply:
Ahh, my bad. Nah, totes fine. Ignore :)